app/assets/javascripts/components/shipment.js.coffee

```diff
@@ -34,7 +34,13 @@ window.Shipment = class Shipment extends React.Component
       td({}
         data.purpose_code + ' - ' + data.source_code + ' - ' + data.year
       )
-      unless @state.changesHistory
+      if @state.changesHistory
+        [
```

Comment:
And if someone has a very bad long name, like me, we could shorten them to initials - "1 change by AF" should be enough to give people a clue, and they can download the csv with full name in it.